internal/dag/serviceapis_processor.go

```diff
@@ -13,9 +13,18 @@
 
 package dag
 
+import (
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	serviceapis "sigs.k8s.io/service-apis/apis/v1alpha1"
+)
+
 // ServiceAPIsProcessor translates Service API types into DAG
 // objects and adds them to the DAG.
 type ServiceAPIsProcessor struct {
+	logrus.FieldLogger
```

Comment:
All the usages of the FieldLogger (p.Error,p.Errorf...) panic, as this has seems to not be initialized (nil).

Reply:
👍 I added this to the init of the processor.